File: sdks/kotlin/src/main/kotlin/com/puxbay/models/Expense.kt
package com.puxbay.models

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class Expense(
    val id: String,
    val category: String,
    val description: String,
    val amount: Double,
    val branch: String,
    @SerialName("receipt_url") val receiptUrl: String? = null,
    val date: String
)

File: sdks/rust/src/models/expense.rs
use serde::{Deserialize, Serialize};

#[derive(Debug, Serialize, Deserialize)]
pub struct Expense {
    pub id: String,
    pub category: String,
    pub description: String,
    pub amount: f64,
    pub branch: String,
    pub receipt_url: Option<String>,
    pub date: String,
}
